src/Internal/Deserialization/FromBytes.purs

```diff
@@ -1,155 +1,135 @@
 module Ctl.Internal.Deserialization.FromBytes
   ( class FromBytes
-  , FromBytesError
-  , _fromBytesError
-  , fromBytesError
   , fromBytes'
   , fromBytes
   , fromBytesEffect
   ) where
 
 import Prelude
 
+import Ctl.Internal.Deserialization.Error (FromBytesError, fromBytesErrorHelper)
 import Ctl.Internal.Error (E)
-import Ctl.Internal.FfiHelpers (ErrorFfiHelper, errorHelper)
+import Ctl.Internal.FfiHelpers (ErrorFfiHelper)
+import Ctl.Internal.Serialization.Address (Address)
+import Ctl.Internal.Serialization.Hash (Ed25519KeyHash, ScriptHash, VRFKeyHash)
 import Ctl.Internal.Serialization.Types
-  ( DataHash
+  ( AuxiliaryDataHash
+  , DataHash
   , Ed25519Signature
+  , GenesisDelegateHash
+  , GenesisHash
   , Mint
   , NativeScript
   , PlutusData
+  , PoolMetadataHash
   , PublicKey
+  , Redeemers
+  , ScriptDataHash
   , Transaction
+  , TransactionBody
   , TransactionHash
+  , TransactionOutput
   , TransactionUnspentOutput
   , TransactionWitnessSet
-  , VRFKeyHash
   , Value
   )
 import Ctl.Internal.Types.ByteArray (ByteArray)
-import Data.Either (Either(Left), hush)
+import Ctl.Internal.Types.CborBytes (CborBytes)
+import Data.Either (hush)
 import Data.Maybe (Maybe(Just, Nothing))
-import Data.Variant (inj)
+import Data.Newtype (unwrap)
 import Effect (Effect)
 import Effect.Exception (throw)
-import Type.Prelude (Proxy(Proxy))
 import Type.Row (type (+))
 
 -- | Calls `from_bytes` method for the appropriate type
 class FromBytes a where
   fromBytes' :: forall (r :: Row Type). ByteArray -> E (FromBytesError + r) a
 
-instance FromBytes DataHash where
-  fromBytes' = _fromBytesDataHash eh
+instance FromBytes Address where
+  fromBytes' = _fromBytes "Address" fromBytesErrorHelper
```

Comment:
Why do we need to explicitly pass `fromBytesErrorHelper` to all these calls to `_fromBytes`?

Reply:
That's a good point, we can avoid repetition, I'll make the change